worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -47,6 +47,26 @@ function onBeforeProcessDocument(e) {
     e.cancel = ! eval(action.conditionFunction)(e.document, arguments);
 }
 
+function onProcessDocument(e) {
+    if (!e.application.getInputMessageProcessor().getProcessSubdocumentsSeparately()) {
```

Comment:
- Flip the `if` statement
- There is no need to use accessors in Javascript - i.e. you can just do:

        if (e.application.inputMessageProcessor.processSubdocumentsSeparately) {